samples/VanillaJSTestApp/index_blankPageRedirectUri.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Quickstart for MSAL JS</title>
```

Comment:
Can you make this unique for this sample?

Reply:
Done